packages/core-app-api/alpha-api-report.md

```diff
@@ -5,32 +5,18 @@
 ```ts
 import { AppTranslationApi } from '@backstage/core-plugin-api/alpha';
 import { i18n } from 'i18next';
+import { TranslationMessages } from '@backstage/core-plugin-api/alpha';
 import { TranslationRef } from '@backstage/core-plugin-api/alpha';
+import { TranslationResource } from '@backstage/core-plugin-api/alpha';
 
 // @alpha (undocumented)
 export class AppTranslationApiImpl implements AppTranslationApi {
   // (undocumented)
-  addLazyResources<Messages extends Record<string, string>>(
-    translationRef: TranslationRef<Messages>,
-    initResources?: Record<
-      string,
-      () => Promise<{
-        messages: TranslationMessages<TranslationRef>;
-      }>
-    >,
-  ): void;
+  addLazyResources(resource: TranslationResource): void;
```

Comment:
```suggestion
  addLazyResource(resource: TranslationResource): void;
```

should this be singular?

Reply:
Yeah but I opted for minimizing changes, looking to remove it completely in a followup.